test/client_test.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "./test_helper"
+require_relative "../src/clients/kdump"
+ 
+describe Yast::KdumpClient do
+ 
+  describe ".cmdKdumpStartup" do
+    let(:alloc_mem_high) {"100"}
+    let(:alloc_mem_low) {"50"}
+    
+    before(:each) do
+      subject.cmdKdumpStartup(options)
```

Comment:
I see that you're trying to reduce redundancy. But I would prefer to write this call in every "it" block because it makes clear what you're testing.